app/usecases/score.py

```diff
@@ -200,3 +189,100 @@ async def build_full_replay(score: Score) -> Optional[BinaryWriter]:
         .write_raw(raw_data)
         .write_i64_le(score.id)
     )
+
+
+class FirstPlaceRow(NamedTuple):
```

Comment:
would kinda like TypedDict to be consistent with rest of codebase but meh

Reply:
changed to `TypedDict`